src/devices/Bmp180/CalibrationData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Bmp180
+{
+    internal class CalibrationData

```

Comment:
Can this be a struct instead?

Reply:
Yes can be, we will need to change the BMP280 CalibrationData